### PR TITLE
(fix)03-2577 Results tree view should be collapsed by default

### PR DESCRIPTION
**Pull request by gitcliff**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR changes the tree view to not expand the first row of the results tree ie "Complete Blood Count" should be minimized under Hematology under Test Results in the patient chart. The rational is to ensure the tree remains unexpanded on load and not the opposite as seen below..
Lab Results Viewer UI improvements needed

## Screenshots
Screenshot 2023-12-04 at 1 25 48 PM

## Related Issue
https://openmrs.atlassian.net/browse/O3-2577

### CODE REVIEW

File: packages/esm-patient-labs-app/src/config-schema.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const configSchema = {
     _default: [
       {
         conceptUuid: 'ae485e65-2e3f-4297-b35e-c818bbefe894',
-        defaultOpen: true,
+        defaultOpen: false,
```
Comment (pirupius) [inline packages/esm-patient-labs-app/src/config-schema.ts:19]: @denniskigen do u have any objections to this? if not let's get it in.

**Review by pirupius — APPROVED** [review]

**Comment by ibacher** [comment]
Why do we want a component that displays nothing by default? I'm having a hard time understanding the rationale for the ask/ Without some kind of rationale, I'm not inclined to merge this in. I appreciate that this is probably the result of discussions I have not been involved in; that rationale should be documented as part of the ticket. Similarly, while not critical, the ticket should be mentioned in the commit to enable us to properly understand this change in the future.

**Comment by caseyi** [comment]
Hi @ibacher thank you for the questions, here is a link to the issue. https://issues.openmrs.org/projects/O3/issues/O3-2577

**Comment by caseyi** [comment]
This change was requested by Grace and is pinned in the Lab Squad Epic

File: packages/esm-patient-labs-app/src/config-schema.ts (hunk 1 above)
Comment (vasharma05) [reply packages/esm-patient-labs-app/src/config-schema.ts:19]: Hi @pirupius , the change looks good to go

**Pull request merged**